src/components/button/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Could we generate those files during build, and have a generic `package.json`?  Do you plan on having differences other than the component name in these?

This could be a phase 2.